gordo_components/builder/build_model.py

```diff
@@ -150,10 +169,62 @@ def build_model(
     }
 
     metadata["model"].update(_get_metadata(model))
-
     return model, metadata
 
 
+def get_metrics_dict(metrics_list, y):
+    """
+        Given a list of metrics that accept a true_y and pred_y as inputs this returns
+        a dictionary with keys in the form '{score}-{tag_name}' for each given target tag
```

Comment:
_very_ mild, but you have an extra indent here in the docs.

Also, as we talked, maybe it is more generic to call things `feature(s)` rather than tags? I'm not terribly concerned about this though...